index.html

```diff
@@ -1,258 +1,469 @@
-                    class="fa fa-github-square fa-2x icon-link icon-link-github" aria-hidden="true"></i></a>
-                <a href="https://www.linkedin.com/in/adi008/" target="_blank"><i
-                    class="fa fa-linkedin-square fa-2x icon-link icon-link-linkedin" aria-hidden="true"></i></a>
-              </p>
-            </div>
-          </div>
-        </div>
-      </div>
-      <div class="col-md-4">
-        <div class="profile-section">
-          <div class="text-center">
-            <img src="./assets/images/Sandeep Velayudhan.jpeg" alt="Sandeep Velayudhan" class="profile-image">
-            <hr>
-          </div>
-          <div class="profile-bio pt-2">
-            <div class="profile-name">Sandeep Velayudhan</div>
-            <div class="profile-skills"> security researcher | open source contributor</div>
-            <hr>
-            <div class="profile-github pt-2">
-              <p>
-                <a href="https://github.com/Sanu1999" target="_blank"><i
-                    class="fa fa-github-square fa-2x icon-link icon-link-github" aria-hidden="true"></i></a>
-              </p>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hacktoberfest 2022</title>
+    <!-- stylesheet -->
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <!-- bootstrap 4.6 cdn -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css"
+      integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N"
+      crossorigin="anonymous"
+    />
+    <!-- google fonts: Roboto -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <!-- font awesome cdn -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <!-- Favicon -->
+    <link rel="icon" type="image/x-icon" href="./assets/images/favicon.png" />
+    <style>
+      body {
+        padding: 25px;
+        background-color: white;
+        color: black;
+        font-size: 25px;
+      }
+      .dark-mode {
+        background-color: black;
+        color: white;
+      }
+      .light-mode {
+        background-color: white;
+        color: black;
+      }
+    </style>
+  </head>
+  <body>
+    <h3 id="DarkModetext">Dark Mode is OFF</h3>
+    <button onclick="darkMode()">Darkmode</button>
+    <button onclick="lightMode()">LightMode</button>
+    <script>

```

Comment:
Hey @Abhinav-3359 !
Add this javascript in `assets/javascript/app.js` file and include it in the `index.html` file.